pkg/lang/capabilities.go

```diff
@@ -2,33 +2,38 @@ package lang
 
 import (
 	"fmt"
-	"io"
-	"regexp"
-
 	"github.com/klothoplatform/klotho/pkg/annotation"
 	"github.com/klothoplatform/klotho/pkg/core"
 	"github.com/klothoplatform/klotho/pkg/multierr"
 	"github.com/klothoplatform/klotho/pkg/query"
 	"github.com/pkg/errors"
 	sitter "github.com/smacker/go-tree-sitter"
+	"io"
```

Comment:
nit: Is this an IDE change? I generally prefer the imports to be std lib > empty line > external libs > empty line > internal packages (which I believe VS Code auto-formats this via the gopls language server).

Reply:
yeah, it looks like `gofmt` (intellij default) vs `goimports` -- which I can change on my end.